packages/orchestrator-ui-components/src/components/WFOTable/utils/withSortableAndFilterableValues.ts

```diff
@@ -0,0 +1,23 @@
+import { WFOTableColumns, WFOTableDataColumnConfig } from './columns';
+
+export function withSortableAndFilterableValues<T>(
+    tableColumnConfig: WFOTableColumns<T>,
+    sortableFieldNames: string[],
+    filterableFieldNames: string[],
+): WFOTableColumns<T> {
+    const tableConfigValues =
+        Object.values<WFOTableDataColumnConfig<T, keyof T>>(tableColumnConfig);
+
+    const updatedTableConfigValues = tableConfigValues.map((value) => ({
```

Comment:
It feels like this can be l11 and l17 can be done at the same. I'm not sure about how exactly though :)

Reply:
It could, but it might be harder to follow what is happening